content/docs/2.6/scalers/apache-kafka.md

```diff
@@ -33,7 +36,27 @@ triggers:
 
 - `bootstrapServers` - Comma separated list of Kafka brokers "hostname:port" to connect to for bootstrap.
 - `consumerGroup` - Name of the consumer group used for checking the offset on the topic and processing the related lag.
-- `topic` - Name of the topic on which processing the offset lag.
+- `topic` - Name of the topic on which processing the offset lag. (Optional)
+
+  > **Note:**
```

Comment:
Could you please move this Note down below the list of parameters? The note itself is great.

Reply:
moved.